src/commands/apps-android-sha-delete.ts

```diff
@@ -0,0 +1,41 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+
+import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
+import { deleteAppAndroidSha } from "../management/apps";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+
+async function initiateAppAndroidShaDeletion(
+  projectId: string,
+  appId: string,
+  shaId: string
+): Promise<void> {
+  const spinner = ora("Deleting Android SHA certificate hash").start();
+
+  try {
+    await deleteAppAndroidSha(projectId, appId, shaId);
+    spinner.succeed();
+  } catch (err) {
+    spinner.fail();
+    throw err;
+  }
+}
+
+module.exports = new Command("apps:android:sha:delete <appId> <shaId>")
+  .description("delete a SHA certificate hash for a given app id.")
+  .before(requireAuth)
+  .action(
+    async (appId: string = "", shaId: string = "", options: any): Promise<void> => {
+      const projectId = getProjectId(options);
+
+      logger.info(
```

Comment:
Same two comments here as on the file above.

Reply:
ok with following message?
`Deleting Android SHA certificate hash with SHA id 5c79a38c3e3a44ac and Android app Id 1:696982949839:android:e35b690950338b48f64cba`

Reply:
Yeah sounds good!